packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1596,6 +1645,42 @@ export class TransactionController extends BaseController<
       });
     });
   }
+
+  /**
+   * Compares and adds history entry to the provided transactionMeta history.
+   *
+   * @param transactionMeta - TransactionMeta to add history entry to.
+   * @param note - Note to add to history entry.
+   */
+  private updateTransactionHistory(
```

Comment:
For the sake of encapsulation and modularity, can we move both of these private methods to `history-utils` also?

Reply:
True, that's nothing to do either with state or another instance code.

Reply:
Fixed in https://github.com/MetaMask/core/pull/1657/commits/bc9d48716cad4b94029cb9318b4feb7c32243e64